tests/test_vjp.py

```diff
@@ -25,6 +25,9 @@
 if not ld._CPP_BINARY_AVAILABLE:
     pytest.skip("No binary module found. Skipping.", allow_module_level=True)
 
+if ld._new_API:
```

Comment:
Can we remove that too?

Reply:
The device does not support VJP computation yet. Is that something we want in this PR?

Reply:
Update: Will open separate PR to integrate VJP support with the device.